index.bs

```diff
@@ -3292,6 +3325,43 @@ The sample code for generating and registering a new key follows:
       });
 </pre>
 
+## Registration Specifically with Platform Authenticator ## {#sample-registration-with-platform-authenticator}
+
+This is flow for when the [=[RP]=] is specifically interested in creating a public key credential with
+a [=platform authenticator=].
+
+1. The user visits example.com. At this point, the user is likely already logged in.
+
+1. The [=[RP]=] script runs the code snippet below.
+
+1. The user agent asks the user whether they are willing to register with the [=[RP]=] using an available [=platform authenticator=].
+
+1. If the user is not willing, terminate this flow.
+
+1. The user is shown appropriate UI and guided in creating a credential using one of the available platform authenticators.
+    Upon successful credential creation, the RP script conveys the new credential to the server.
+
+<pre class="example" highlight="js">
+    if (!PublicKeyCredential) { /* Platform not capable of the API. Handle error. */ }
+
+    navigator.credentials.isPlatformAuthenticatorAvailable()
```

Comment:
Is someone crafting a new PR with which to propose a resolution to this discussion?